README.md

```diff
@@ -41,3 +39,76 @@ Everyday
+
+In the [HZA Research Guide](https://github.com/CodeforHawaii/Hawaii-Zoning-Atlas/wiki/Research-Guide) The user can find the starting point for the map which is the [datasheet](https://docs.google.com/spreadsheets/d/1YGt_Y70oy6qc09ZZ7kip9DM2JGtRC2fAHxi6JXOIsSk/edit#gid=0).
+
+This datasheet is manually populated by people going through the various zoning laws (The links for the zoning laws are found in the repo's research guide.) and filling out cells according to the guidelines laid out in [How to Make a Zoning Atlas](https://www.zoningatlas.org/how).
+
+After the data sheet is populated, it is then exported to a csv where the following steps need to occur:
+  * The county tabs are combined together
+  * Validation is run on the data
+  * The final csv is given the name "hawaii-zoning-data.csv"
+
+The hawaii-zoning-data.csv file should then be placed into the data-pipeline folder.
+
+In the data-pipeline folder there is a "gis" folder that holds the output of processing various shape files. These shape files come from the maps given in the [HZA Research Guide](https://github.com/CodeforHawaii/Hawaii-Zoning-Atlas/wiki/Research-Guide). The method of processing the shape files is outlined in [How to Make a Zoning Atlas](https://www.zoningatlas.org/how).
+
+#### Data Processing
+
+Once the data has been aggregated and in the correct form, it is processed by the CombineJurisdictions Jupyter Notebook. The notebook exports the final.geojson file that is placed in the repository's "data" directory. This is the data the populates the website.
+
+#### Automation Goals
+
+[GitHub actions](https://github.com/features/actions) is the tool we are using to automate the various parts of the data processing pipeline. 
+
+Different GitHub actions will perform the following tasks:
+* Pull the data from the datasheet into a csv that combines the counties
+* Run validation to ensure that the data is correct (prevent mistypings, blank lines, etc)
+* Commit the validated and changed files to the repository 
+
+```mermaid
+sequenceDiagram
+    participant Actions
+    participant Spreadsheet
+
+    Actions->>Spreadsheet: Pull data for all the counties (Maui, Oahu, Big Island, Kauai)
+
+    loop Validation
+        Actions->>Actions: Validate the pulled data against a schema (prevent mistypings, blank lines, ...etc)
+    end
+
+    loop Commit
+        Actions->>Actions: Commit the validated and changed files to this repository
+    end
+```
+
+Once the data is ready for processing, another set of actions will perform the following tasks:
+* Run the data through the CombineJurisdiction notebook
+* Run an action to minimize the final.geojson file
+* Commit the final.geojson file to the repository 
+  * The name of the final.geojson should include a timestamp and a reference to the original data used to generate it. This will allow us to rollback to different versions if needed.
```

Comment:
```suggetion
  * The name of the `final.geojson` should include a timestamp and a reference to the original data used to generate it. This will allow us to rollback to different versions if needed.
```